exporter/prometheusremotewriteexporter/exporter_concurrency_test.go

```diff
@@ -0,0 +1,153 @@
+//go:build !race
+// +build !race
+
+// note: this test doesn't pass currently due to a race condition in batchTimeSeries
```

Comment:
@ArthurSens I discovered a race condition when multiple goroutines run which seems unrelated to my change. The batching logic uses shared state which won't work with multiple goroutines. I added some build tags above to exclude this test until it's fixed.

Reply:
Hmmm, I've run your test on the main branch, and indeed, we have a data race like you said :O 

I am not sure how to proceed here. To be honest, I think we need to fix the test first, so we're safe to merge this one, too

Reply:
I've opened a PR trying to address the issue: https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/36524

Wanna share your opinion? :)